CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Rename AWS signing script.
 - Update descriptions/comments in scripts.
 - standalone/Open Fortune 500 websites in a browser.zst - Fix typo in `http://www,pbfenergy.com`
+- Update export_session script to copy multiple zap files created during log file rotation
```

Comment:
Should not be added here, the scan hooks are not part of the add-on.

Reply:
Removed incorrect line from changelog.